server/app/views/admin/programs/ProgramFormBuilder.java

```diff
@@ -51,47 +54,52 @@ private static FormTag buildProgramForm(
       boolean editExistingProgram) {
     FormTag formTag = form().withMethod("POST");
     formTag.with(
+        requiredFieldsExplanationContent(),
         h2("Internal program information"),
         h3("This will only be visible to administrators"),
         FieldWithLabel.input()
             .setId("program-name-input")
             .setFieldName("adminName")
-            .setLabelText("Enter internal name or nickname of this program")
+            .setLabelText("Enter internal name or nickname of this program *")
```

Comment:
not blocking, personal preference: WDYT about removing the space before the asterisk? I feel like I typically see it right after the text (e.g. mocks: https://docs.google.com/presentation/d/1v0RpMkAt85Jvl9418CTGQm0iBpvjyvzuTo5ONlymBgE/edit?resourcekey=0-K0QNTzv28d03QPg9pOJZ_Q#slide=id.g12c549fe21d_1_11)

Reply:
Good call. Done!